website/app/components/doc/icons-list/item.hbs

```diff
@@ -18,5 +18,10 @@
     <Doc::MetaRow @label="Ember" @valueToCopy={{@meta.iconName}} @copyable={{true}} />
     <Doc::MetaRow @label="React" @valueToCopy={{@meta.name}} @copyable={{true}} />
     <Doc::MetaRow @label="Keywords" @valueToShow={{@meta.description}} />
+    <a
```

Comment:
there should be a way to ember-ify this so it doesn't cause the whole app to reload when this link is clicked. I'll see if i can find it.

Reply:
i got the link to be formed but clicking it doesn't refresh the page correctly 😭

Reply:
I spent a few more minutes looking at this and couldn't quite figure it out. I think the way it works now is ok though and we can always revisit later.

Reply:
I did the same, and I don't understand why it doesn't work 🤔
Maybe something we can play with at the next HDS engineering sync?

Reply:
👍  what I saw was that clicking the link updated the query params but it had no affect on the tracked property that was watching that param. It may be somewhat related to the unusual way these pages are wired up because everything shares the same route yet the "model" in this page's case is actually just reading directly from the flight-icons npm package so there is no model to "refresh" when the query params changes 🤷

Reply:
@Dhaulagiri I think we can go live with this version, and in a separate PR try to figure out how to overcome this issue.

Reply:
Happy to back it out, but the diff we worked on was really close so I pushed up a working version here to get this in here. Have a look and let me know if you think it doesn't work.